brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/brooklyn/spec/ProvisioningPropertiesModifierTest.java

```diff
@@ -30,10 +46,11 @@
 
     @Test(dataProvider = "mapping")
     public void testSetsProperty(String templateProperty, ConfigKey<?> configKey, String value) {
-        nodeTemplate.setProperties(ImmutableMap.<String, AbstractPropertyValue>of(
-                templateProperty, new ScalarPropertyValue(value)));
-        ProvisioningPropertiesModifier builder = new ProvisioningPropertiesModifier(mgmt);
-        builder.apply(testSpec, nodeTemplate, topology);
+        Mockito.when(alien4CloudFacade.resolveProperty(
+                Mockito.anyString(), Mockito.any(ToscaApplication.class), Mockito.anyString())).thenReturn(value);
+        Mockito.when(toscaApplication.getNodeTemplate(Mockito.anyString())).thenReturn(nodeTemplate);
+        ProvisioningPropertiesModifier builder = new ProvisioningPropertiesModifier(mgmt, alien4CloudFacade);
```

Comment:
Likewise, should test resolving properties in Alien4CloudFacadeTest now that it doesn't happen here.